lib/devices/mimic.ml

```diff
@@ -9,11 +8,11 @@ type mimic = Mimic
 let mimic = typ Mimic
 
 let mimic_happy_eyeballs =
-  let packages = [ package "mimic-happy-eyeballs" ~min:"0.0.5" ] in
+  let packages = [ package "mimic-happy-eyeballs" ~min:"0.0.8" ] in
```

Comment:
just as a reminder to my future self -- here earlier mirage version do not restrict the mimic-happy-eyeballs upper bound, thus 0.0.8 could be used... but given that mimic-happy-eyeballs 0.0.8 requires dns-client 8.0.0 (and earlier mimic-happy-eyeballs require dns-client < 8), this is fine.

in general, I prefer if we put upper bounds on all the packages to allow future development of these packages. It is not clear to me what the versioning scheme of mimic-happy-eyeballs is - thus whenever its `Make` functor changes (or `connect`), will the major version be increased? or the minor? at the moment there are only patch-level releases - but for mirage it'd be very nice to have such a versioning scheme where we can then put adequate upper bounds in the tool.